Tools/cases_generator/generate_cases.py

```diff
@@ -403,6 +404,12 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
 
             self.write_provenance_header()
 
+            self.out.emit("\n" + textwrap.dedent("""
+                #ifndef Py_BUILD_CORE
+                #  error "this header requires Py_BUILD_CORE define"
+                #endif
+            """).strip())
```

Comment:
Why bother asking for a review if you're going to merge it while I'm getting coffee? What response time do you expect for reviews?

Reply:
Yeah,  I was venting at @vstinner who ignored that and merged **eight minutes later**. :-(

Reply:
I didn't ask for a review on such "cleanup" change. I didn't see that @corona10 asked for a review.